src/modelbench/consistency_checker.py

```diff
@@ -0,0 +1,304 @@
+import json
+import os
+from abc import ABC, abstractmethod
+from collections import Counter, defaultdict
+from itertools import product
+from tabulate import tabulate
+from typing import Dict, List
+
+from modelbench.run_journal import journal_reader
+
+LINE_WIDTH = os.get_terminal_size().columns
+
+
+class JournalSearch:
```

Comment:
I really like this approach.

Reply:
Thanks! I think this will make it easy to independently optimize the journal querying later on.